testdata/streams/TestTestPublishEventsBatchGZIP_gzip_put_error.golden

```diff
@@ -0,0 +1,58 @@
+batch 0
```

Comment:
Is this file being used?

Reply:
yes its a golden file used for tests.
https://medium.com/@jarifibrahim/golden-files-why-you-should-use-them-47087ec994bf